index.rst

```diff
@@ -6,7 +6,158 @@ Roadmap for Community Alert Filters with the ANTARES Broker
+
+Scope
+-----
+
+The Rubin CST will:
+
+#. Incorporate the Rubin science community’s input to develop, implement, test, and validate up to 20 Community Alert Filters in the ANTARES broker, starting with an initial set of ~5 filters during commissioning.
+#. Provide documentation and use-case examples for the community filters.
+#. Maintain and add to the set of community filters during Rubin Operations.
+
+These community filters will:
+
+* be designed to prevent redundancy among user-defined filters,
+* together as a set, serve a broad range of science interests,
+* individually, each meet a well-defined science goal,
+* significantly down-select the number of alerts per visit,
+* be subject to technical constraints, and
+* be maintained throughout Rubin operations.
+
+In order to serve a broad range of science interests, these filters should be simple to understand and prioritize completeness over purity.
+For example, they would serve to down-select alerts for further real-time investigation by users (e.g., follow-up observations); they would not serve to define pure samples of unique astrophysical objects for population analysis.
+
+Timeline
+--------
+
+This timeline guides the work of the Rubin Community Science team.
+
+FY25 Q1, Oct-Dec 2024:
+
+* prepare and release this tech note
+* learn by implementing a few simple filters in ANTARES with ZTF
+
+FY25 Q2, Jan-Mar 2025:
+
+* prepare and advertise call input from the community
+* review the input and draft ~5 initial community filters
+
+FY25 Q3, Apr-Jun 2025 (or when LSST alerts begin):
+
+* implement ~5 initial community filters
+* release documentation and use-case tutorials
+
+FY25 Q4, Jul-Sep 2025:
+
+* prepare validation report and ‘lessons learned’ for the community
+
+FY26 Q1, Oct-Dec 2026:
+
+* second round of community input for additional community filters
+
+
+Technical constraints
+=====================
+
+The following can be considered as rough guidelines.
+
+Alert contents
+--------------
+
+See Section 3.5 of the Rubin Data Products Definitions Document (`LSE-163 <https://lse-163.lsst.io/>`_) for a list of alert packet contents, and `sdm-schemas.lsst.io/apdb.html <https://sdm-schemas.lsst.io/apdb.html>`_ has the as-implemented column names.
+
+The community filters will use only the contents of the LSST alert packet plus any additional or derived data added by ANTARES, such as cross-matches with external (non-Rubin) catalogs performed by ANTARES.
+
+The contents of the alert packets might evolve over Rubin Operations, thereby enabling new community filters.
+For example, the final set of time series features to be computed during Alert Production remains to be determined (`DMTN-118 <https://dmtn-118.lsst.io/>`_).
+
+Once a Rubin data release exists (DP2, DR1), the community filters will be able to query and use proprietary LSST data, such as multi-year light curves or the photo-z of potential host galaxies.
```

Comment:
```suggestion
Once a Rubin data release exists (DP2, DR1), a Rubin data rights holder will be able to query and use proprietary LSST data, such as multi-year light curves or the photo-z of potential host galaxies to continue analysis of data returned by the community filters .
```

Reply:
This entire sentence, "Once a Rubin data release exists ... returned by the community filters." was removed from the roadmap.